packages/grant-explorer/src/features/discovery/LandingPage.tsx

```diff
@@ -49,35 +47,6 @@ const LandingPage = () => {
             ))}
         </div>
       </LandingSection>
-      <LandingSection
-        title="Apply for funding"
-        action={<ViewAllLink to="/rounds?status=apply">View all</ViewAllLink>}
-      >
-        <div className="flex gap-8 items-center">
-          <div className="hidden md:block md:w-1/3 space-y-8">
-            <p className="text-2xl">
-              Bring your project to life with Gitcoin's vibrant ecosystem of
-              public goods funding opportunities.
-            </p>
-            <p className="text-2xl">
-              Discover new grant rounds currently accepting applications, and
-              apply for funding today!
-            </p>
-          </div>
-          <div className="grid md:grid-cols-2 gap-x-6 w-full md:w-2/3">
-            {(roundsTakingApplications.data ?? createRoundLoadingData(4))
-              .slice(0, 4)
-              ?.map((round) => (
-                <div key={round?.id}>
-                  <RoundCard
-                    round={round}
-                    isLoading={roundsTakingApplications.isLoading}
-                  />
-                </div>
-              ))}
-          </div>
```

Comment:
Was this removal unintentional?

Reply:
https://github.com/gitcoinco/grants-stack/pull/2521#issuecomment-1790657264

Reply:
Ah I see